### PR TITLE
included get all tags documentation 

### PR DESCRIPTION
**Pull request by beblicarl**
#### Pre-Submission Checklist

-   [x] Your pull request targets the `main` branch.
-   [x] Branch name is descriptive and follows the guidelines [here](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices).
-   [x] All tests pass . Use `git commit --amend` to amend any fixes.

#### Checklist:

-   [x] All tests are passing.
-   [x] Tested changes locally.
-   [x] Improved documentation or readme where necessary.
-   [x] Your commit messages format follows our commit message guideline [here](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices#commits).

#### What does this PR do?

This PR includes the get all tags documentation
#### How should this be manually tested?

1. Start the server with` npm run dev`
2. Copy and paste this link in the browser to see the get all tags endpoint documentation `http://localhost:3000/docs/#/tag/getTags`

#### What are the relevant issues?
We had to add the get all tags endpoint documentation so it can be understood and used

### CODE REVIEW

File: docs/swagger.yaml [hunk 1]
```diff
@@ -1520,6 +1520,42 @@ paths:
...
+      summary: Get all tags
+      description: Get all tags
+      responses:
+        '200':
+          description: A successful operation
+          content: 
+            application/json:
+              schema:
+                type: object
+                properties:
```
Comment (itzgeebee) [inline docs/swagger.yaml:1536]: @beblicarl since we're getting all tags, is it supposed to be an object or array of objects?

Comment (beblicarl) [reply docs/swagger.yaml:1536]: you are right, i have implemented and pushed the changes

**Review by itzgeebee — COMMENTED** [review]
@jocrah I am done reviewing in case you want to take a look

**Comment by jocrah** [comment]
> @jocrah I am done reviewing in case you want to take a look

thanks @itzgeebee, kindly proceed

**Review by itzgeebee — APPROVED** [review]

**Pull request merged**